src/web/components/SlotBodyEnd.tsx

```diff
@@ -9,6 +9,8 @@ import {
 import { useApiFn } from '../lib/api';
 import { useVisibility } from '../lib/useVisibility';
 
+const ophanEventRegister: { [key: string]: boolean } = {};
```

Comment:
The idea with this in-memory registry is that we're keeping track of which events we've sent to Ophan already for _this_ page view. This will all be reset on the next page view.